1-exercises/B-object-arrays/exercise-1.js

```diff
@@ -13,8 +13,13 @@ let people = [
   { name: "Elise", age: 35 },
 ];
 
-let peopleNames =
+let peopleNames = people.map(name => {
```

Comment:
Deago, I trust that this works but it looks unorthodox.
There really is no need to use a 'for in' loop.
One 'map' will do. Have a look at the solution by CYF